Generate-GitBranches.ps1

```diff
@@ -248,7 +248,7 @@ LAYERED_SIZE=$( if ($kv.Contains('LAYERED_SIZE')) { $kv['LAYERED_SIZE'] } else {
 
         "Creating .gitignore" | Write-Host -ForegroundColor Green
         @"
-*
+/*
```

Comment:
no, this ignores only root

Reply:
this is the full syntax, same as `*`

Reply:
behaviorally its the same, because if the parent is ignored all children are as well, and if parent is excluded from ignore all children are also excluded from ignore. I guess it may be fine.

Reply:
it's the exact equivalent of `*`